packages/typespec-autorest/src/openapi.ts

```diff
@@ -458,6 +458,7 @@ export async function getOpenAPIForService(
         prop.type.kind === "Scalar" &&
         ignoreDiagnostics(
           program.checker.isTypeAssignableTo(
+            // eslint-disable-next-line @typescript-eslint/no-deprecated
```

Comment:
What will projectionBase have at this point, shouldn't this just be prop.type?

Reply:
Projection base was to make it work with std scalars but with mutators I preventing mutating the compiler types so it's unnecessary